scripts/simplify_network.py

```diff
@@ -715,7 +715,8 @@ def merge_isolated_nodes(n, threshold, aggregation_strategies=dict()):
 
     n = pypsa.Network(snakemake.input.network)
 
-    linetype = snakemake.config["lines"]["types"][380.0]
+    base_voltage = max(snakemake.config["electricity"]["voltages"])
```

Comment:
Instead of doing this, let's make the base_voltage **explicit** in all configs.

From:
```
electricity:
  voltages: [220., 300., 380.]
```

To:
```
electricity:
  base_voltage: [380.]
  voltages: [220., 300., 380.]
```

So you can read in the script `snakemake.config["electricity"]["base_voltage"]` instead of `max(snakemake.config["electricity"]["voltages"])`

Reply:
Maybe `base_voltage: 380.`?
A list there may be an overcomplication

Reply:
Agreeing with Davide 👍🏽